static/app/views/releases/list/index.tsx

```diff
@@ -120,11 +120,12 @@ class ReleasesList extends DeprecatedAsyncView<Props, State> {
         'releases', // stateKey
         `/organizations/${organization.slug}/releases/`, // endpoint
         {query}, // params
-        {disableEntireQuery: true}, // options
+        {disableEntireQuery: true}, // options - prevent cursor from being passed into query
```

Comment:
Just a nit: Add a todo for the time pagination gets added, I think you already have a github issue too to link to it.

Reply:
Oh - the api actually doesn't need to be paginated at all - i'll make that more clear in the PR description